.github/workflows/test.yml

```
@@ -48,3 +48,60 @@ jobs:
 
       - name: Run widget tests, unit tests
         run: flutter test --machine --coverage
+
+  generate_mason_bundle:
+    # The job generate_mason_bundle identify that the job test must complete successfully
+    # before this generate_mason_bundle job will run
+    needs: test
+    name: Generate Mason bundle
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
```

@nmint8m I think this particular task should only proceed if the previous task was completed successfully
```suggestion
    runs-on: ubuntu-latest
    timeout-minutes: 30
    if: success()
```

---

@nmint8m Did you miss this review?

---

@manh-t @luongvo According to [the document](https://docs.github.com/en/actions/using-jobs/using-jobs-in-a-workflow#defining-prerequisite-jobs) that I attached:

> Use `jobs.<job_id>.needs` to identify any jobs that must complete successfully before this job will run. It can be a string or array of strings. If a job fails or is skipped, all jobs that need it are skipped unless the jobs use a conditional expression that causes the job to continue. If a run contains a series of jobs that need each other, a failure or skip applies to all jobs in the dependency chain from the point of failure or skip onwards.

I think using `jobs.<job_id>.needs` is enough, no need to use `if: success()`

---

@nmint8m Thanks for clarifying 👍